tests/integration-tc/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="
+  http://maven.apache.org/POM/4.0.0
+  http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.pulsar.tests</groupId>
+        <artifactId>tests-parent</artifactId>
+        <version>2.2.0-incubating-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.apache.pulsar.tests</groupId>
+    <artifactId>integration-tc</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Apache Pulsar :: Tests :: Container Based integration tests</name>
+
+    <properties>
+        <testcontainers.version>1.8.0</testcontainers.version>
+        <commons-compress.version>1.15</commons-compress.version>
+    </properties>
+
+    <dependencies>
```

Comment:
you should add a dependency of `tests/docker-images/latest-version-image`. so this integration is running after the docker image is built.